tests/unit/utils/test_config.py

```diff
@@ -7,17 +7,28 @@
 
 
 @pytest.mark.parametrize(
-    "yaml_content, expected_config",
+    "json_content, expected_config",
     [
         (
+            # Given: multiple models, two variables
+            # Expected: Config object with two models, no variables
             """
-            models:
-              - name: model1
-                deployment_id: dep1
-                temperature: 0.0
-              - name: model2
-                deployment_id: dep2
-                temperature: 0.5
+            {
+                "VARIABLE_NAME": "value",
+                "VARIABLE_NAME2": "value2",
+                "models": [
+                    {
+                        "name": "model1",
+                        "deployment_id": "dep1",
+                        "temperature": 0.0
+                    },
+                    {
+                        "name": "model2",
+                        "deployment_id": "dep2",
+                        "temperature": 0.5
+                    }
+                ]
+            }
             """,
             Config(
                 models=[
```

Comment:
please add the the other variables to expected config too,
                "VARIABLE_NAME": "value",
                "VARIABLE_NAME2": "value2",

Reply:
This test expects only the model configuration to be returned. The env variables are set in `settings.py` function `load_env_from_json()`

Reply:
added a unit test for `load_env_from_json()`